Source/SwiftLintFramework/Rules/Style/LetVarWhitespaceRule.swift

```diff
@@ -231,7 +232,9 @@ private extension SwiftDeclarationKind {
 private extension SwiftLintFile {
     // Zero based line number for specified byte offset
     func line(byteOffset: Int) -> Int {
-        guard let line = contents.bridge().lineAndCharacter(forByteOffset: byteOffset)?.line else { return -1 }
-        return line - 1
+        let lineIndex = contents.bridge().lines().firstIndexAssumingSorted { line in
```

Comment:
Could probably make this even faster by reusing `lines()` instead of computing every time.

Reply:
Yeah. just found this one.